docker/gitea/versions.yaml

```diff
@@ -0,0 +1,29 @@
+# Copyright 2022 Google LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+
+versions:
+  - dir: '1/debian11/1.16/gitea'
+    repo: 'gitea1'
+    tags:
+    - '1.16-debian11'
+    - '1.16'
+    - '1-debian11'
+    - '1'
+    - 'latest'
```

Comment:
Missing patch version tags. I see [here](https://dl.gitea.io/gitea) that 1.16 has several patches.